sal_interfaces/HVAC/HVAC_Commands.xml

```diff
@@ -3,68 +3,39 @@
 <SALCommandSet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://lsst-sal.tuc.noao.edu/schema/SALCommandSet.xsd">
   <SALCommand>
     <Subsystem>HVAC</Subsystem>
-    <EFDB_Topic>HVAC_command_chiller01P01_enable</EFDB_Topic>
-    <Description>Enable command for the chiller01P01 device.</Description>
+    <EFDB_Topic>HVAC_command_enable_device</EFDB_Topic>
+    <Description>Enable command for a single HVAC device.</Description>
     <item>
-      <EFDB_Name>comandoEncendido</EFDB_Name>
-      <Description>Command Started</Description>
-      <IDL_Type>boolean</IDL_Type>
+      <EFDB_Name>device_id</EFDB_Name>
+      <Description>The ID indicating which single device needs to be enabled. The IDs can be found in the DeviceId enumeration.</Description>
```

Comment:
I think "single" is redundant here. I realize you originally coded a bitmask, but that version was never released, and DeviceId is not a bitmask, so I don't think anyone will be confused.

Reply:
Good suggestion. I removed it here and also for the enable and disable commands.